_posts/2021-05-21-dask-distributed-user-journey.md

```diff
@@ -0,0 +1,164 @@
+
+The platforms your organisation has will be the result of many arbitrary technology choices. What programming languages does your company use? What deals did vendors offer at the time of procurement? What skills do the current IT staff have? What did your CTO have for breakfast the day they chose a vendor?
+
+Within Dask we don't care which platform decisions your organisations make and try to build deployment tools for as many popular platforms as we can including:
+
+- Hadoop via [dask-yarn](https://github.com/dask/dask-yarn)
+- Kubernetes via [dask-kubernetes](https://github.com/dask/dask-kubernetes) and the [helm chart](https://github.com/dask/helm-chart)
+- HPC (with schedulers like SLURM, PBS and SGE) via [dask-jobqueue](https://github.com/dask/dask-jobqueue)
+- Cloud platforms (including AWS, Azure and GCP) with [dask-cloudprovider](https://github.com/dask/dask-cloudprovider)
+
+As a user within these organisations you may have been onboarded to one of these platforms. You've probably been given some credentials and a little training on how to launch jobs on it.
+
+The `dask-foo` tools listed above are designed to sit on top of those platforms and submit jobs on your behalf as if they were individual compute jobs. But instead of submitting a Python script to the platform we submit Dask schedulers and workers and then connect to them to leverage the provisioned resource. Clusters on top of clusters.
+
+With this approach your IT team has full control over the compute resource. They can ensure folks get their fair share with quotas and queues. But you as a user gets the same Dask experience you are used to on your local machine.
+
+Your data may be in a slightly different place on these platforms though. Perhaps you are on the cloud and your data is in object storage for example. Thanks to tools built on [fsspec](https://filesystem-spec.readthedocs.io/en/latest/) like [s3fs](https://github.com/dask/s3fs) or [adlfs](https://pypi.org/project/adlfs/) we can read this data in pretty much the same way. So still not much change to your workflow.
+
+```python
+from dask.distributed import Client
+from dask_cloudprovider.azure import AzureVMCluster
+import dask.dataframe as dd
+
+cluster = AzureVMCluster(resource_group="<resource group>",
+                         vnet="<vnet>",
+                         security_group="<security group>",
+                         n_workers=1)
+client = Client(cluster)
+
+df = dd.read_csv("adl://.../2018-*-*.csv")
+df.groupby(df.account_id).balance.sum().compute()
+```
+
+## Centralizing your Dask resources
+
+When your organisation gets enough folks adopting and using Dask it may be time for your IT team to step in and provide you with a managed service. Having many users submitting many ad-hoc clusters in a myriad of ways is likely to be less efficient than a centralised managed and more importantly ordained service from IT.
```

Comment:
I don't think it's clear here about why the alternative approach is less efficient in several ways, or why it's important to have it as an official IT service.

I'd like to see a suggestion on where the threshold is for moving from `dask-jobqueue`/`dask-foo`/etc. to Dask Gateway. That way users can be confident about which "tier" of dask deployment options suits them at the moment.